ReactAndroid/src/androidTest/java/com/facebook/react/tests/CatalystNativeJSToJavaParametersTestCase.java

```diff
@@ -299,9 +305,28 @@ public void testGetTypeFromArray() {
     assertEquals(ReadableType.Null, array.getType(6));
   }
 
+  public void testCustomTypeViaArgumentExtractor() {
+    mCatalystInstance.getJSModule(TestJSToJavaParametersModule.class).returnCustomType();
```

Comment:
Land failed as tests don't compile since `TestJSToJavaParametersModule#returnCustomType` isn't defined. Make sure to run the integration tests first: `./scripts/run-android-local-integration-tests.sh`